server/controllers/finance/subsidy.js

```diff
@@ -30,7 +30,10 @@ function list(req, res, next) {
   'use strict';
 
   var sql =
-   'SELECT id, account_id, label, description, value, created_at, updated_at FROM subsidy';
+   'SELECT subsidy.id, subsidy.account_id, subsidy.label, subsidy.description, subsidy.value, subsidy.created_at, ' +
```

Comment:
In general, `.list()` functions should only return the minimal amount of information.  In this case, it should probably return only `id`, `label`, and `value`.

If you need access to more properties, implement a `detailed=1` query string that will send back the full table.